rnd/autogpt_builder/src/hooks/useAgentGraph.ts

```diff
@@ -637,11 +643,8 @@ export default function useAgentGraph(
         nodes: formattedNodes,
         links: links,
       };
-
-      if (savedAgent && deepEquals(payload, savedAgent)) {
-        console.debug(
-          "No need to save: Graph is the same as version on server",
-        );
+      if (savedAgent && deepEquals(payload, savedAgent, true)) {
+        console.warn("No need to save: Graph is the same as version on server");
```

Comment:
This isn't a `warn` level message

Reply:
Unless we fix the UX for saving, not having this warning is super confusing.